src/main/pages/dashboard.page.js

```diff
@@ -2,10 +2,18 @@ const { I, testData } = inject();
 
 const lblDashboardH3 = { css: '.ant-typography' }
 
+const menuProjects = '.ant-menu-title-content > [href $= "projects"]'
```

Comment:
can provide any example for xpath? @nganlk-setel 
cc @tuantransetel @antuking

Reply:
Hi @nhuthm-setel updated